BZ#5224

===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/ig_NG,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- libc/localedata/locales/ig_NG	2007/09/30 16:47:13	1.1
+++ libc/localedata/locales/ig_NG	2007/10/28 08:58:19	1.2
@@ -117,6 +117,11 @@
 collating-element <o_2'> from "<U1ECD><U0301>"
 collating-element <O_2`> from "<U1ECC><U0300>"
 collating-element <o_2`> from "<U1ECD><U0300>"
+collating-symbol  <s-h-ig>
+collating-element <S-H> from "<U0053><U0048>"
+collating-element <S-h> from "<U0053><U0068>"
+collating-element <s-H> from "<U0073><U0048>"
+collating-element <s-h> from "<U0073><U0068>"
 collating-symbol  <ubelowdot>
 collating-element <U_>  from "<U0055><U0323>"
 collating-element <u_>  from "<U0075><U0323>"
